packages/sui-domain/src/fetcher/AxiosFetcher.js

```diff
@@ -9,6 +9,57 @@ export default class AxiosFetcher {
    */
   constructor({config}) {
     this._axios = axios.create(config)
+    this._initInterceptors()
+  }
+
+  /**
+   * Initializes interceptors
+   */
+  _initInterceptors() {
+    this._axios.interceptors.response.use(
+      this._interceptResponse,
+      this._interceptError
+    )
+  }
+
+  /**
+   * Intercepts all responses and, in case an interceptor has been defined, calls it
+   * @param {Object} response
+   * @returns {Object}
+   */
+  _interceptResponse(response) {
+    if (AxiosFetcher.responseInterceptor !== undefined)
```

Comment:
should you be checking for `typeof AxiosFetcher.responseInterceptor === 'function'`?

Reply:
I think we could add that to improve stability. However, `responseInterceptor` should only contain functions, defined by calling the `setResponseInterceptor` method.

However, if this `PR` is finally accepted, I'll add the extra type-checking before-merging.

Thank you!